client/src/featureform/serving.py

```diff
@@ -49,7 +50,10 @@ def check_feature_type(features):
         if isinstance(feature, tuple):
             checked_features.append(feature)
         elif isinstance(feature, str):
+            # TODO: Need to identify how to pull the run id
             checked_features.append((feature, "default"))
+        elif isinstance(feature, ff.register.FeatureColumnResource):
```

Comment:
@ahmadnazeri, should we be pulled in `FeatureColumnResource` more directly rather than pulling in `featureform as ff`?